services/venice-server/src/main/java/com/linkedin/venice/listener/StorageResponseHandlerCallback.java

```diff
@@ -0,0 +1,19 @@
+package com.linkedin.venice.listener;
+
+import com.linkedin.davinci.listener.response.ReadResponse;
+import com.linkedin.venice.response.VeniceReadResponseStatus;
+
+
+/**
+ * Callback interface to handle the response from the {@link StorageReadRequestHandler#processRequest} method.
+ */
+public interface StorageResponseHandlerCallback {
```

Comment:
That's very good. I was hoping we could do something like this to reduce the copy pasted code, though the outline of the needed abstraction was not clear to me. Thanks for putting it together!

Reply:
Thanks! 🙏